README.md

```diff
@@ -93,6 +93,12 @@ This guide will cover:
 - Join a member cluster to `karmada` control plane.
 - Propagate an application by `karmada`.
 
+### Demo
+
+There are several demonstrations of common cases.
+
+[![asciicast](https://asciinema.org/a/407612.svg)](https://asciinema.org/a/407612)
```

Comment:
Does this will jump into asciinema.org?

Reply:
Yeah, It does. Let me change another way to embed the demo.

Reply:
[karmada  website](https://karmada.io/) maybe it's a place to put the demo.

Reply:
> Yeah, It does. Let me change another way to embed the demo.

Done

Reply:
> [karmada website](https://karmada.io/) maybe it's a place to put the demo.

It's ok to save in this repo. No plan shown on the website yet.